renovation_core_graphql/utils/types/root.graphql

```diff
@@ -0,0 +1,10 @@
+type SearchLinkResult{
+    description: String
+    value: String
+}
+
+scalar SearchLinkFilterInput
```

Comment:
@e-lobo If you would please add a comment so it appears in the documentation about the structure of the JSON String

Reply:
@assemmarwan updated 👍